# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: toolkit/components/downloads/DownloadCore.jsm
# Commit: 74f99033251c
# Full Hash: 74f99033251cd6c36eb6b0b5a4713eb6b5c793fa
# Author: Jonathan Kingston <jkt@mozilla.com>
# Date: 2019-02-13 09:48:26
# Regressor Bug: 1520868
# File Overlap Count: 1
# Description:
#   Bug 1520868 - Replacing AsyncOpen2 with AsyncOpen always r=valentin
#   
#   Replacing js and text occurences of asyncOpen2
#   Replacing open2 with open
#   
# ==============================================================================

diff -r 3378fd5f4f22 -r 74f99033251c toolkit/components/downloads/DownloadCore.jsm
--- a/toolkit/components/downloads/DownloadCore.jsm	Tue Feb 12 17:58:19 2019 +0100
+++ b/toolkit/components/downloads/DownloadCore.jsm	Tue Feb 12 16:08:25 2019 +0000
@@ -1962,7 +1962,7 @@
 
       // Open the channel, directing output to the background file saver.
       backgroundFileSaver.QueryInterface(Ci.nsIStreamListener);
-      channel.asyncOpen2({
+      channel.asyncOpen({
         onStartRequest: function(aRequest, aContext) {
           backgroundFileSaver.onStartRequest(aRequest, aContext);
 